src/AzureBatch/docker/tti_image/Dockerfile

```diff
@@ -1,14 +1,18 @@
 FROM devito_azure_base:v1.0
 
 # Install storage queue
-RUN pip3 install azure-storage-blob h5py segyio boto3
-RUN pip3 install azure-storage
+RUN pip3 install azure-storage-blob==2.1.0 h5py segyio boto3
 
 # Current sympy bug -> force version 1.5
 RUN pip3 install sympy==1.5
 
-# Add application
-ADD ./tti /app/tti
+# Add devito setup from JUDI, keep only Python part
+RUN git clone --single-branch --branch examples https://github.com/slimgroup/JUDI.jl /app/JUDI.jl \
```

Comment:
The scripts above assume it installed in `~/.julia/dev/JUDI`

Reply:
Not sure which scripts you mean?

Reply:
I updated the script to look for the module in `/app/pysource`

Reply:
Oh ok cool